tests/src/test/scala/com/nvidia/spark/rapids/HashAggregatesSuite.scala

```diff
@@ -75,7 +75,7 @@ class HashAggregatesSuite extends SparkQueryCompareTestSuite {
     val options = FuzzerOptions(asciiStringsOnly = true, numbersAsStrings = false,
         maxStringLen = 4)
     val schema = FuzzerUtils.createSchema(Seq(DataTypes.StringType, DataTypes.IntegerType))
-    FuzzerUtils.generateDataFrame(spark, schema, 100, options, seed = 0)
```

Comment:
I personally think that there are a number of places where having an option set that matches the default value is good. This is one of those places. Perhaps we should just not have a default value for seed then, so it forces the user to set it.